introduction.md

```diff
@@ -1,2 +1,8 @@
 Hi, I'm Pablo.
 My favorite Code School path is the Ruby one.
+
+Deadly skills
+-------------
+* Ruby
+* JavaScript
+* Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?